setup.py

```diff
@@ -89,30 +88,6 @@ def is_requirement(line):
     return line and line.strip() and not line.startswith(('-r', '#', '-e', 'git+', '-c'))
 
 
-class Tox(TestCommand):
```

Comment:
`setuptools.command.test` is removed in setuptools 72 and I couldn't find any usage of this class.